src/lsg/patterns/panes/preview-pane/index.tsx

```diff
@@ -4,13 +4,14 @@ import * as React from 'react';
 import styled from 'styled-components';
 
 export interface PreviewPaneProps {
-	handleMouseDownLeft?: React.MouseEventHandler<HTMLElement>;
-	handleMouseDownRight?: React.MouseEventHandler<HTMLElement>;
-	handleMouseMove?: React.MouseEventHandler<HTMLElement>;
-	handleMouseUp?: React.MouseEventHandler<HTMLElement>;
+	id?: string;
+	onMouseDownLeft?: React.MouseEventHandler<HTMLElement>;
```

Comment:
We had a discussion on when to use "handle", and when to use "on", but I forgot the outcome. @LKuechler ?

Reply:
To my knowledge the React community consensus is `handle*` for internal methods and `on*` for property names.